NEWS.md

```diff
@@ -23,6 +23,8 @@ deprecation notice (#1229)
 
 ## Documentation
 
+- Fixed `derive_var_disposition_status` argument to render correctly (https://pharmaverse.github.io/admiral/reference/derive_var_disposition_status.html)
```

Comment:
Please put a set of parentheses, i.e. `()`, after the function name and put the correpsonging issue number rather than the website link at the end.

Reply:
updated